demos/fonts/all.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <div id="outputBravura"></div>
+    <div id="outputGonville"></div>
+    <div id="outputPetaluma"></div>
+    <!-- The standard vexflow build includes three music fonts. -->
+    <script src="../../build/vexflow.js"></script>
+    <script type="module">
+      console.log('VexFlow BUILD ' + Vex.Flow.BUILD);
+
+      ////////////////////////////////////////////////////////////////////////
```

Comment:
Not a huge fan of this big comment line :-)

Reply:
Sounds good. Will remove. Sometimes I put in these "horizontal separators" to help guide the code reader between large code blocks, but here I have extra braces { } anyways, so they're probably not needed.